jdk/src/share/classes/java/util/stream/SliceOps.java

```diff
@@ -160,29 +160,7 @@ private static long calcSliceFence(long skip, long limit) {
             <P_IN> Node<T> opEvaluateParallel(PipelineHelper<T> helper,
```

Comment:
Should we remove these functions instead of throwing?

Reply:
Well, I _could_ remove them... (probably...). But I think I was concerned about deleting methods from public Java interfaces.

Reply:
Isn't that better in general than throwing an exception? If someone's developing a contract we probably don't want to have this exposed on the public api